File: represent/bench/c_flat/SVCOMP_0079__SVCOMP__ExSymExeTestAssignments__ExSymExeTestAssignments.c
int f(int x);
int test(int x);

int f(int x) {

    if (x < 0) return 1;
    return test(x);
}

int test(int x) {

    x = 3;
    if (x > 0)
    else {
      return 0;
    }
    return 1;
}

File: represent/bench/java_flat/SVCOMP_0079__SVCOMP__ExSymExeTestAssignments__ExSymExeTestAssignments.java
class Main {
  int field;

  public static boolean f (int x) {
    if (x < 0) return true;
    return test(x);
  }

  public static boolean test(int x) {
    x = 3;
    if (x > 0) System.out.println("branch BOO1");
    else {
      return false;
    }
    return true;
  }
}
